collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -86,6 +87,9 @@
     private final DriveHelper driveHelper;
     private final GoogleAccountsManager accountsManager;
 
+    private static final String altitudeTitlePostfix = "-altitude";
+    private static final String accuracyTitlePostfix = "-accuracy";
```

Comment:
In the whole code, we use capital letters and `_` between words for static final fields so I think it should be good to keep it up.

Reply:
Agreed. For more: https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html